Patches/MoreMechanoids/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -313,19 +319,23 @@
                 <li Class="PatchOperationReplace">
                     <xpath>Defs/ThingDef[defName="Mech_Assaulter"]/statBases/ArmorRating_Sharp</xpath>
                     <value>
-                        <ArmorRating_Sharp>15</ArmorRating_Sharp>
+                        <ArmorRating_Sharp>24</ArmorRating_Sharp>
```

Comment:
This is awfully high--higher than the centipede or normal quality power armor on an infantry-sized mechanoid. I'd reduce this to the range of 16 - 18 sharp. If this is attempting to simulate the "shield" it carries, this would be a good opportunity to use partial coverage so that the entire thing doesn't have this much armor.